src/main/java/learnositysdk/request/Init.java

```diff
@@ -449,5 +448,55 @@ private void validateSecurityPacket (Object securityPacket) throws Exception
             Date date = new Date();
             this.securityPacket.put("timestamp", dateFormat.format(date));
         }
-    }   
+    }
+
+    /**
+     * Disables telemetry.
+     *
+     * We use telemetry to enable better support and feature planning. It is therefore not advised to
```

Comment:
That's what we need in the README! (Also for PHP q:)

Reply:
I've got this on my to-do list now.